src/videojs-contrib-hls.js

```diff
@@ -387,6 +387,7 @@ class HlsHandler extends Component {
   setOptions_() {
     // defaults
     this.options_.withCredentials = this.options_.withCredentials || false;
+    this.options_.blacklistDuration = this.options_.blacklistDuration || 5 * 60;
```

Comment:
Probably also want to make sure it is a number and use options passed to player.src like bandwidth below